### ISSUE TITLE
Newly updated ncu informs of update

### ISSUE THREAD

**Issue by krokofant** [tjunnone/npm-check-updates#372]
- [x] I have read the list of [known issues](https://github.com/tjunnone/npm-check-updates#known-issues) before filing this issue.
- [x] I have searched for [similiar issues](https://github.com/tjunnone/npm-check-updates/issues) before filing this issue.

* node version: v8.1.2
* npm version: 5.0.4
* npm-check-updates version: 2.12.1

---------------------------

I often react to a newly updated ncu reporting that the *just installed updated* is available to update to. I find it hard to replicate but this has come to my mind before so I thought I'd share the issue.

**Comment by krokofant** [comment]
@raineorshine It's when ncu itself is updated.

Example output:
```bash
✘  ~  npm install -g npm-check-updates
/usr/local/bin/npm-check-updates -> /usr/local/lib/node_modules/npm-check-updates/bin/npm-check-updates
/usr/local/bin/ncu -> /usr/local/lib/node_modules/npm-check-updates/bin/ncu
+ npm-check-updates@2.12.1
added 45 packages, removed 3 packages and updated 20 packages in 16.285s
 ~  ncu -g


   ╭──────────────────────────────────────────────╮
   │                                              │
   │      Update available 2.11.3 → 2.12.1        │
   │   Run npm i -g npm-check-updates to update   │
   │                                              │
   ╰──────────────────────────────────────────────╯

```

Next time ncu is run it does not report about the update.

**Comment by raineorshine** [comment]
That's really strange. That is from a drop-in module that should work out
of the box. Hard to investigate since it involves a version upgrade.

**Comment by krokofant** [comment]
I found the issue upstream https://github.com/yeoman/update-notifier/issues/67

**Comment by raineorshine** [comment]
Good find!!! Thank you! I will watch that issue closely.
